src/components/include/cos_defkernel_api.h

```diff
@@ -56,6 +56,10 @@ struct cos_compinfo *cos_compinfo_get(struct cos_defcompinfo *defci);
  * cos_sched_aep_get: returns the sched aep info from the defcompinfo.
  */
 struct cos_aep_info *cos_sched_aep_get(struct cos_defcompinfo *defci);
+/*
+ * boot_info: leverages the above calls to conveniently fetch the current components cos_compinfo struct
+ */
+struct cos_compinfo *boot_info(void);
```

Comment:
Got rid of this too in my merge of your code! Not just from cos_defkernel_api, but really not using this function.